SKILL.tsv

```diff
@@ -4525,15 +4525,15 @@ SKILL_20150714_004524	$Increases the damage dealt on an enemy with [Bounce Shot]
 SKILL_20150714_004525	$Knockbacks enemies hit by [Barrage] with a 10% chance per attribute level.
 SKILL_20150714_004526	$Inflicts [Slow] on enemies hit by [Bounce Shot] with a 10% chance per attribute level.
 SKILL_20150714_004527	$Spiral Arrow: Weakness Attack
-SKILL_20150714_004528	The attacks with the effect of [Spiral Arrow] will be Critical with a certain probability to the enemies that are in Armor Break status. The probability will increase by 10% per Attribute level.
+SKILL_20150714_004528	$Has a chance to apply Critical to attacks with the effect of [Spiral Arrow] to enemies afflicted with Armor Break. Increases the chance by 10% per attribute level.
 SKILL_20150714_004529	$Increases the damage dealt on an enemy with [Scatter Caltrop] by 1% per attribute level.
 SKILL_20150714_004530	$Increases the damage dealt on an enemy with [Stone Shot] by 1% per attribute level.
 SKILL_20150714_004531	$Increases the damage dealt on an enemy with [Rapid Fire] by 1% per attribute level.
-SKILL_20150714_004532	$Increases the damage dealt on an enemy with [Tear Down] by 1% per attribute level.
+SKILL_20150714_004532	$Increases the damage dealt on an enemy with [Teardown] by 1% per attribute level.
 SKILL_20150714_004533	$Increases the duration of [Scatter Caltrop] by 1 second per attribute level.
-SKILL_20150714_004534	When stepped on [Scatter Caltrops], the target will fall into [Bleeding] with 8% probability per Attribute level.
-SKILL_20150714_004535	Tear Down: Tear down the sculpture
-SKILL_20150714_004536	[Tear Down] enables you to tear down the fragments of Divine Tree.
+SKILL_20150714_004534	$There is a 8% chance per attribute level to afflict [Bleed] when stepping on [Scatter Caltrop].
```

Comment:
"There is a 8% chance per attribute level for the target to be inflicted with [Bleed] when stepping on [Scatter Caltrops]", your version makes it sound as if the target will be afflicted with bleed if _you_ step on the caltrops...